examples/viewer.py

```diff
@@ -73,6 +94,59 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
             key.Z: "move_up",
         }
 
+        # Load a TrueTypeFont plugin and open the font file
+        self.display_font = text.FontManager().load_and_instantiate("TrueTypeFont")
+        relative_path_to_font = "../data/fonts/ProggyClean.ttf"
+        self.display_font.open_file(
+            os.path.join(os.path.dirname(__file__), relative_path_to_font),
+            180,
+        )
+
+        # Glyphs we need to render everything
+        self.glyph_cache = text.GlyphCache(mn.Vector2i(2048))
+        self.display_font.fill_glyph_cache(
+            self.glyph_cache,
+            string.ascii_lowercase
+            + string.ascii_uppercase
+            + string.digits
+            + ":-_+,.! %µ",
+        )
+
+        # magnum text object that displays CPU/GPU usage data in the app window
+        self.window_text = text.Renderer2D(
+            self.display_font,
+            self.glyph_cache,
+            HabitatSimInteractiveViewer.DISPLAY_FONT_SIZE,
+            text.Alignment.TOP_LEFT,
+        )
+        self.window_text.reserve(HabitatSimInteractiveViewer.MAX_DISPLAY_TEXT_CHARS)
+
+        # text object transform in window space is Projection matrix times Translation Matrix
+        self.window_text_transform = mn.Matrix3.projection(
+            mn.Vector2(self.viewport_size)
+        ) @ mn.Matrix3.translation(
+            mn.Vector2(
+                self.viewport_size[0]
+                * -HabitatSimInteractiveViewer.TEXT_DELTA_FROM_CENTER,
+                self.viewport_size[1]
+                * HabitatSimInteractiveViewer.TEXT_DELTA_FROM_CENTER,
+            )
+        )
+        self.shader = shaders.DistanceFieldVectorGL2D()
```

Comment:
```suggestion
        self.shader = shaders.VectorGL2D()
```

Since you aren't really using any outline anyway, let's use a simpler shader. (The same is used in the `viewer.cpp`.)